modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -21,6 +21,18 @@ image::mruntime-scatter-gather.png[Diagram of Scatter-Gather Component]
 
 WARNING: Configure your Scatter-Gather component to have at least two routes; otherwise, your Mule application throws an exception and does not start.
 
+== Result
+
+The Result Message after a scatter gather of each route is stored in a result map in the form
+`{"0":messageFromRoute0,"1":messageFromRoute1,...}`
+
+To extract all resulting payloads to an array you can use
+
```

Comment:
TODO: let's get verification of this script on the output.

Reply:
As this is just a sniped which may be used for set-payload or set-variable or in a more complex dataweave I think header (%dw) and output type can be omitted here.